avocado/plugins/multiplex.py

```diff
@@ -35,23 +34,28 @@ def __init__(self, *args, **kwargs):
 
     def configure(self, parser):
         parser = super(Multiplex, self).configure(parser)
+        parser.add_argument("--summary", type=int, help="Verbosity of "
```

Comment:
An interface (either a custom enum or reusing numbers for logging.*) as a guideline to plugin writers would be really helpful.  That will ensure some level of consistency among plugins.